Makefile

```diff
@@ -120,6 +127,10 @@ lintfix: my.env .docker-build
 run: my.env .docker-build
 	${DC} up web
 
+.PHONY: runwebdebug
+runwebdebug: my.env .docker-build
+	${DC} run --rm --service-ports web shell python ichnaea/webapp/app.py
```

Comment:
Doesn't `make run` already do this? It runs `web` which runs `docker/run_web.sh` which sees `LOCAL_DEV_ENV=true` and runs the app the way you're suggesting here.

Reply:
The difference is when I add a ``pdb`` breakpoint. For example, when I add ``import pdb; pdb.set_trace()`` to ``ichnaea/content/views.py``, ``homepage_view``, and refresh http://localhost:8000, I get a 500 Internal Server Error with ``make run``, and an interactive ``pdb`` session with ``make runwebdebug``.

Also, it seems useful to shutdown the ``web`` container when running this, otherwise it needs to be done manually.

Reply:
I think I'd rather have a `webshell` make rule like this:

```
.PHONY: webshell
webshell: my.env .docker-build
    ${DC} run --rm --service-ports web shell
```

That gets you into a web context and covers running the web server with pdb and some other things.

Thinking ahead, I want to redo how static assets are built, so then the `node.make` thing will go away. This will be useful for that, too.